client/lib/wp/handlers/README.md

```diff
@@ -0,0 +1,45 @@
+http-evelope-normalizer
+=======================
+
+This handler-wrapper catch errors from the server-side response generating an
+error object when the body has error information.
```

Comment:
I think the readme could use more background about `http_envelope` like you've given in the description here. Otherwise it's not clear why we have to do this.